examples/kubernetes/deployment+service.privileged.yaml

```diff
@@ -28,6 +28,22 @@ spec:
         - /certs/cert.pem
         - --tlskey
         - /certs/key.pem
+        readinessProbe:
+          exec:
+            command:
+            - sh 
+            - -c 
+            - "buildctl debug workers"
+          initialDelaySeconds: 5
+          periodSeconds: 5
+        livenessProbe:
+          exec:
+            command:
+            - sh 
```

Comment:
no need to use `sh -c`?

Reply:
I tried without ...
"exec: \"buildctl debug workers\": executable file not found in $PATH": unknown

Reply:
```yaml
command:
- buildctl
- debug
- workers
```